### PR TITLE
Feature/course info editing

### PR DESCRIPTION
**Pull request by rglasgow-deloitte**
Implements the first iteration of metadata editing.

### CODE REVIEW

File: src/components/Dashboard/CourseData/CourseDataContainer.test.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+const testData = {
+  unique_record_identifier: "000e893d-5741-4c07-8dd8-2e3d9fa4b862",
+  date_deleted: null,
+  date_inserted: "2021-07-20T19:34:31.567535Z",
+  date_transmitted: null,
+  metadata: {
+    Metadata_Ledger: {
+      Course: {
+        CourseURL:
+          "https://jkodirect.jten.mil/html/COI.xhtml?course_prefix=CES&course_number=-103",
```
Comment (jdiouf) [inline src/components/Dashboard/CourseData/CourseDataContainer.test.js:21]: Let's replace the data with fake values

File: src/components/Dashboard/CourseData/CourseDataContainer.test.js [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+  it("does render", async () => {
+    await act(async () => {
+      // mocking the calls for jest
+      axios.get.mockImplementationOnce(() =>
+        Promise.resolve({ data: testData })
+      );
+      render(
+        <MemoryRouter
+          initialEntries={[
+            "/dashboard/JKO/course/000e893d-5741-4c07-8dd8-2e3d9fa4b862",
```
Comment (jdiouf) [inline src/components/Dashboard/CourseData/CourseDataContainer.test.js:67]: should use fake values

File: src/components/Dashboard/CourseData/CourseDataContainer.test.js [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+  it("does render correct data for each data field", async () => {
+    await act(async () => {
+      // mocking the calls for jest
+      axios.get.mockImplementationOnce(() =>
+        Promise.resolve({ data: testData })
+      );
+      render(
+        <MemoryRouter
+          initialEntries={[
+            "/dashboard/JKO/course/000e893d-5741-4c07-8dd8-2e3d9fa4b862",
```
Comment (jdiouf) [inline src/components/Dashboard/CourseData/CourseDataContainer.test.js:92]: fake values

File: src/components/Dashboard/CourseData/CourseDataContainer.test.js [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+  it("does show error message", async () => {
+    await act(async () => {
+      // mocking the calls for jest
+      axios.get.mockImplementationOnce(() =>
+        Promise.resolve({ data: testData })
+      );
+      render(
+        <MemoryRouter
+          initialEntries={[
+            "/dashboard/JKO/course/000e893d-5741-4c07-8dd8-2e3d9fa4b862",
```
Comment (jdiouf) [inline src/components/Dashboard/CourseData/CourseDataContainer.test.js:125]: fake values

File: src/components/Dashboard/CourseData/CourseDataContainer.test.js (hunk 1 above)
Comment (rglasgow-deloitte) [reply src/components/Dashboard/CourseData/CourseDataContainer.test.js:21]: done

File: src/components/Dashboard/CourseData/CourseDataContainer.test.js (hunk 2 above)
Comment (rglasgow-deloitte) [reply src/components/Dashboard/CourseData/CourseDataContainer.test.js:67]: done

File: src/components/Dashboard/CourseData/CourseDataContainer.test.js (hunk 3 above)
Comment (rglasgow-deloitte) [reply src/components/Dashboard/CourseData/CourseDataContainer.test.js:92]: done

File: src/components/Dashboard/CourseData/CourseDataContainer.test.js (hunk 4 above)
Comment (rglasgow-deloitte) [reply src/components/Dashboard/CourseData/CourseDataContainer.test.js:125]: done

**Review by jdiouf — APPROVED** [review]

**Review by schittlur — APPROVED** [review]

**Pull request merged**